File: rest/src/main/java/io/sparkled/rest/service/easingtype/EasingTypeRestServiceHandler.java
package io.sparkled.rest.service.easingtype;

import io.sparkled.model.animation.easing.reference.EasingTypes;
import io.sparkled.rest.service.RestServiceHandler;

import javax.ws.rs.core.Response;

public class EasingTypeRestServiceHandler extends RestServiceHandler {

    public Response get() {
        return getJsonResponse(EasingTypes.get());
    }
}

File: rest/src/main/kotlin/io/sparkled/rest/service/easingtype/EasingTypeRestServiceHandler.kt
package io.sparkled.rest.service.easingtype

import io.sparkled.model.animation.easing.reference.EasingTypes
import io.sparkled.rest.service.RestServiceHandler

import javax.ws.rs.core.Response

open class EasingTypeRestServiceHandler : RestServiceHandler() {

    fun get(): Response {
        return respondOk(EasingTypes.get())
    }
}